bitfield/admin.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, field, request, params, model, model_admin, field_path):
 
     def queryset(self, request, queryset):
         filter_kwargs = dict(
-            (p, BitHandler(v, ()))
+            (p, BitHandler(v, getattr(queryset.model, p)))
```

Comment:
Would `getattr` on a user-supplied model a safe operation? I need to take a look at this later...